zerver/webhooks/zapier/doc.html

```diff
@@ -40,3 +40,33 @@
 <p>You're done! Example message:</p>
 
 <img class="screenshot" src="/static/images/integrations/zapier/005.png"/>
+
+<p>
+    <b>Weather Update with Zapier!</b><br />
+</p>
+<p>
+    Go to the following link <code>https://zapier.com/app/explore</code> after you sign into your zapier account<br />
+    Select Send Weather info to your app via webhooks.
+
+    <img class="screenshot" src="/static/images/integrations/zapier/006.png" /><br /><br />
+    Select trigger app as <b>weather by zapier</b><br /><br />
+    <img class="screenshot" src="/static/images/integrations/zapier/007.png" /><br /><br />
+    Next select the option <b>Today's Forecast</b><br /><br />
+    <img class="screenshot" src="/static/images/integrations/zapier/008.png" /><br /><br />
+    Fill in the latitude and longitude of your location<br /><br />
+    <img class="screenshot" src="/static/images/integrations/zapier/009.png" /><br /><br />
+    After selecting <code>Webhook</code> as action choose the  <code>POST</code> option and then fill in the url as <code>https://yourZulipDomain.zulipchat.com/api/v1/external/zapier?api_key=abcdefgh&stream=zapier</code><br /><br />
+    Select Payload type as <code>JSON</code><br /><br />
+</p><br />
+<p>
+    You have to add 2 fields:<br />
+    <ul>
+        <li><code>subject</code> is field corresponding to a subject of the message</li>
+        <li><code>content</code> is field corresponding to a content of the message.Like <code>Summary</code>, <code>Temperature Max</code> and so on</li>
```

Comment:
This should be:
~~~
    <li><code>subject</code> is a field corresponding to the subject of the message, such as <code>Here is your weather update for the day</code>.</li>
    <li><code>content</code> is a field corresponding to the content of the message, such as <code>Summary</code>, <code>Temperature Max</code>, and so on.</li>
~~~